imap_processing/hi/hi_l1a.py

```diff
@@ -54,6 +54,46 @@
 )
 TOTAL_COUNTERS = ("a_total", "b_total", "c_total", "fee_de_recd", "fee_de_sent")
 
+# MEMDMP Packet definition of uint32 fields
```

Comment:
Can you add a comment about what these numbers mean, something like: The offset index (assuming all 4-byte fields) of where these fields are located within the packet